From 407b8adb446122b550e517f37f182a872308d16f Mon Sep 17 00:00:00 2001
From: Harald Welte <laforge@openmoko.org>
Date: Wed, 8 Oct 2008 21:36:10 +0100
Subject: [PATCH] glamo_fb: sync comment with code

the comment claims that DHCLK is enabled, while the
code actually disables it.

No code change, purely cosmetic patch.

Signed-off-by: Harald Welte <laforge@openmoko.org>
---
 drivers/mfd/glamo/glamo-fb.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/mfd/glamo/glamo-fb.c b/drivers/mfd/glamo/glamo-fb.c
index aa901dd..8f5d6b1 100644
--- a/drivers/mfd/glamo/glamo-fb.c
+++ b/drivers/mfd/glamo/glamo-fb.c
@@ -104,7 +104,7 @@ static struct glamo_script glamo_regs[] = {
 	 * no TVCLK, no partial display, hw dest color from fb,
 	 * no partial display mode, LCD1, software flip,  */
 	{ GLAMO_REG_LCD_MODE2, 0x1020 },
-	  /* no video flip, no ptr, no ptr, dhclk,
+	  /* no video flip, no ptr, no ptr, dhclk off,
 	   * normal mode,  no cpuif,
 	   * res, serial msb first, single fb, no fr ctrl,
 	   * cpu if bits all zero, no crc
-- 
1.5.6.5

